main.css

```diff
@@ -14,6 +14,31 @@
     font-size: 20px;
     line-height: 28px;
     color: #172b4d;
+    width: 1440px;
```

Comment:
- [ ] Kindly remove this line as this makes the display much wider than it should be on mobile view.
![image](https://user-images.githubusercontent.com/62290545/161051995-252ab571-874c-4e04-8e09-1ee791802e4d.png)